engine/src/chunk.cpp

```diff
@@ -5673,6 +5673,89 @@ MCTextChunkIterator::MCTextChunkIterator(Chunk_term p_chunk_type, MCStringRef p_
         MCLocaleBreakIteratorRelease(break_iterator);
 }
 
+// AL-2015-02-10: [[ Bug 14532 ]] Add text chunk iterator constructor for restricted range chunk operations.
+MCTextChunkIterator::MCTextChunkIterator(Chunk_term p_chunk_type, MCStringRef p_text, MCRange p_restriction)
+{
+    /* UNCHECKED */ MCStringCopy(p_text, text);
+    type = p_chunk_type;
+    
+    if (type == CT_CHARACTER && (MCStringIsNative(text) || (MCStringIsSimple(text) && MCStringIsUncombined(text))))
```

Comment:
I believe we resolved it by tweaking the ICU break iterators to treat CR LF as two graphemes. Though things may have changed since...

There are other non-automatically-combining BMP character compositions (i.e ones that don't get composed under NFC but do get decomposed via NFKD) like the "ff" and "ffi" ligatures but I'm not sure they are relevant in this case.